app/models/concerns/curation_concerns/characterization.rb

```diff
@@ -0,0 +1,47 @@
+# This module points the FileSet to the location of the technical metdata.
+# By default, the file holding the metadata is :original_file and the terms
+# are listed under ::characterization_terms.
+# Implementations may define their own terms or use a different source file, but
+# any terms must be set on the ::characterization_proxy by the Hydra::Works::CharacterizationService
+#
+# class MyFileSet
+#   include CurationConcerns::FileSetBehavior
+#
+#   self.characterization_proxy = :master_file
+#   self.characterization_terms = [:term1, :term2, :term3]
+#
+# end
+module CurationConcerns
+  module Characterization
+    extend ActiveSupport::Concern
+
+    included do
+      def self.characterization_terms
+        [
+          :format_label, :file_size, :height, :width, :filename, :well_formed,
+          :page_count, :file_title, :last_modified, :original_checksum, :mime_type
+        ]
+      end
+
+      def self.characterization_proxy
+        :original_file
+      end
+
+      delegate(*characterization_terms, to: :characterization_proxy)
+
+      def characterization_proxy
+        return NullCharacterizationProxy.new if send(self.class.characterization_proxy).nil?
+        send(self.class.characterization_proxy)
+      end
+    end
+
+    class NullCharacterizationProxy
+      def method_missing(*_args)
```

Comment:
OK. I was briefly considering suggesting that you loop over `self.characterization_terms` and define those methods rather than use `method_missing` but meh, I think we should merge and move on.